remote-processor/Socket.cpp

```diff
@@ -108,7 +108,7 @@ bool CSocket::read(void* pvData, uint32_t uiSize)
 
     while (uiSize) {
 
-        int32_t iAccessedSize = ::recv(_iSockFd, &pucData[uiOffset], uiSize, MSG_NOSIGNAL);
+        int32_t iAccessedSize = ::recv(_iSockFd, &pucData[uiOffset], uiSize, 0);
```

Comment:
I guess there is no MSG_DEFAULT or similar (could not find anything like that in `man recv`) ?